docs/en/cloud/reference/changelog.md

```diff
@@ -99,8 +99,8 @@ This release brings the public release of the ClickHouse Cloud Programmatic API
 - S3 access using IAM roles. You can now leverage IAM roles to securely access your private Amazon Simple Storage Service (S3) buckets (please contact support to set it up)
 
 ### Scaling changes
-- Horizontal scaling. Workloads that require more parallelization can now be configured with any number of additional replicas (please contact support to set it up)
-- CPU based autoscaling. CPU-bound workloads can now benefit from additional triggers for autoscaling policies
+- [Horizontal scaling](/docs/en/manage/scaling#adding-more-nodes-horizontal-scaling). Workloads that require more parallelization can now be configured with any number of additional replicas (please contact support to set it up)
```

Comment:
Currently we are limiting this to 10 replicas. Either we can say 10 replicas, or just leave it ambiguous and take out the phrase "any number of" from this sentence.